src/syslog_error_h.erl

```diff
@@ -195,7 +195,6 @@ log_crash(false, Pid, Report, State) ->
     Time = calendar:now_to_local_time(os:timestamp()),
     Event = {Time, {error_report, self(), {Pid, crash_report, Report}}},
     Msg = <<<<"[crash] ">>/binary, (iolist_to_binary(sasl_report:format_report(fd, all, Event)))/binary>>,
```

Comment:
`Msg = iolist_to_binary([<<"[crash] ">>|sasl_report:format_report(fd, all, Event)]),`
would avoid one copying of result `sasl_report:format_report/3`